restriction/404.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>404 Not found</title>
+        <link
+            rel="stylesheet"
+            href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.5.0/css/bootstrap.min.css"
+        />
+        <link rel="stylesheet" href="css/404.css" />
+    </head>
+    <body>
+        <!-- tsParticles container -->
+        <div id="tsparticles"></div>
+        <div class="text-center" id="givemelight">
+            <h1>404</h1>
+            <h2>This page does not exist.</h2>
+        </div>
+        
+        <script src="https://cdn.jsdelivr.net/npm/tsparticles@1.41.0/tsparticles.min.js"></script>
+        <script src="https://cdn.jsdelivr.net/npm/tsparticles-interaction-light@1.41.0/tsparticles.interaction.light.min.js"></script>
```

Comment:
This plugin is not use, you can remove it

Reply:
Please review 😄